pkg/templates/hostpid/template.go

```diff
@@ -0,0 +1,39 @@
+package hostpid
+
+import (
+	"fmt"
+
+	"golang.stackrox.io/kube-linter/pkg/check"
+	"golang.stackrox.io/kube-linter/pkg/config"
+	"golang.stackrox.io/kube-linter/pkg/diagnostic"
+	"golang.stackrox.io/kube-linter/pkg/extract"
+	"golang.stackrox.io/kube-linter/pkg/lintcontext"
+	"golang.stackrox.io/kube-linter/pkg/objectkinds"
+	"golang.stackrox.io/kube-linter/pkg/templates"
+	"golang.stackrox.io/kube-linter/pkg/templates/hostpid/internal/params"
+)
+
+func init() {
+	templates.Register(check.Template{
+		HumanName:   "Host PID",
+		Key:         "host-pid",
+		Description: "Flag Pod sharing host's process namespace",
+		SupportedObjectKinds: config.ObjectKindsDesc{
+			ObjectKinds: []string{objectkinds.DeploymentLike},
+		},
+		Parameters:             params.ParamDescs,
+		ParseAndValidateParams: params.ParseAndValidate,
+		Instantiate: params.WrapInstantiateFunc(func(_ params.Params) (check.Func, error) {
+			return func(_ lintcontext.LintContext, object lintcontext.Object) []diagnostic.Diagnostic {
+				podSpec, found := extract.PodSpec(object.K8sObject)
+				if !found {
+					return nil
+				}
+				if podSpec.HostPID {
+					return []diagnostic.Diagnostic{{Message: fmt.Sprintf("resource %s shares,  host's process namespace.", object.K8sObject.GetName())}}
```

Comment:
Again, you don't need to mention the resource's name.
```suggestion
					return []diagnostic.Diagnostic{{Message: "object shares the host's process namespace (via hostPID=true)"}}
```

Reply:
done.